Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Multi-stage Dockerfile intended to minimize production image size
+# Based on https://blog.hasura.io/an-exhaustive-guide-to-writing-dockerfiles-for-node-js-web-apps-bbee6bd2f3c4
+
+# Base node image
+FROM node:carbon AS base
+WORKDIR /app
+
+# Install dependencies
+FROM base AS dependencies
+COPY package.json ./
+COPY package-lock.json ./
+COPY .babelrc ./
+RUN npm install
+
+# Copy files and build
+FROM dependencies AS build
+WORKDIR /app
+COPY server /app/server
+COPY bin/_run-docker /app/bin/_run-docker
+RUN npm run build
+
+# Release with Alpine
```

Comment:
Unfortunately, we are on docker 1.12, so we can't support multi-stage builds, which were introduced in 17.05.

Reply:
If we build on CircleCI, this shouldn't be a problem. Am I missing something? Is this not built on CircleCI?

Reply:
Yes, this is correct. I've confirmed that the images are backwards compatible, so ignore my first comment.